spec/unit/knife/dummy_winrm_connection.rb

```diff
@@ -0,0 +1,20 @@
+module Dummy
+  class WinRMTransport
+    attr_reader :httpcli
+
+    def initialize
+      @httpcli = HTTPClient.new
+    end
+  end
+
+  class Connection
+    attr_reader :transport
+    attr_accessor :logger
+
+    def initialize
+      @transport = WinRMTransport.new
+    end
+
+    def shell; end
+  end
+end
```

Comment:
This file and `spec/support/dummy_winrm_connection.rb` are identical. I'd delete this one.

Reply:
Nuked